packages/client/src/Config.ts

```diff
@@ -123,6 +123,24 @@ export interface ControlLayerConfig {
      * is not formed within this time, the client's network node will throw an error.
      */
     networkConnectivityTimeout?: number
+
+    /**
+     * URL of the autocertifier service used to obtain TLS certificates and subdomain names for the WS server.
+     */
+    autocertifierUrl?: string
```

Comment:
We seem to have a naming convention that `AutoCertifier` consist of two words. Maybe would drop the auto prefix here (or maybe in all names, as there is no non-auto certifier)